requirements.txt

```diff
@@ -1,2 +1,3 @@
 # generated from manifests external_dependencies
 ofxparse
+odoo-test-helper
```

Comment:
see https://github.com/OCA/bank-statement-import/blob/15.0/test-requirements.txt, it should go in test-requirements.txt

Reply:
@MiquelRForgeFlow I changed this now. Thanks!